### PR TITLE
ja: Translate Chapter 60 (Shared State)

### PR DESCRIPTION
**Pull request by kantasv**
Hi, ja translation folks (#652 ). Here's an MR for the chapter "Shared States." Could you review the translations? any feedback would be appreciated. Thank you! 

Note: There are several lines containing outdated `msgid`, but we can refresh the `ja.po` file in a separate MR. We can slightly update corresponding `msgstr` afterwards.

### CODE REVIEW

File: po/ja.po [hunk 1]
```diff
@@ -14084,19 +14084,27 @@ msgid ""
...
 #: src/concurrency/shared_state.md:6
 msgid ""
 "[`Arc<T>`](https://doc.rust-lang.org/std/sync/struct.Arc.html), atomic "
 "reference counted `T`: handles sharing between threads and takes care to "
 "deallocate `T` when the last reference is dropped,"
 msgstr ""
+"[`Arc<T>`](https://doc.rust-lang.org/std/sync/struct.Arc.html), アトミックに"
+"参照カウントする型 （atomic reference counted）を `T` とするとき : スレッド間"
+"の共有を扱い、最後の参照がドロップされたとき `T` をデアロケートすることを担当"
+"する、"
```
Comment (kantasv) [inline po/ja.po:14099]: Translation confidence: Low

File: po/ja.po [hunk 2]
```diff
@@ -14168,38 +14190,49 @@ msgid ""
...
+"struct.Mutex.html#impl-Sync-for-Mutex%3CT%3E) のブランケット実装があることに"
+"注目してください。"
 
 #: src/concurrency/shared_state/mutex.md:31
 msgid ""
 "`Mutex` in Rust looks like a collection with just one element - the "
 "protected data."
 msgstr ""
+"Rustにおける`Mutex`とは、一つの要素 — 保護されたデータ — からなるコレクション"
+"のようなものです。"
```
Comment (kantasv) [inline po/ja.po:14203]: Translation confidence: Low. We normally don't use — (emdash) in Japanese, so we can also paraphrase the line with parenthesis "（）"?

File: po/ja.po [hunk 3]
```diff
@@ -14209,46 +14242,59 @@ msgid ""
 "[`PoisonError`](https://doc.rust-lang.org/std/sync/struct.PoisonError.html). "
 "You can call `into_inner()` on the error to recover the data regardless."
 msgstr ""
+"スレッドにパニックを起こした`Mutex`がある場合、保護すべきデータが整合性の欠け"
+"た状態にある可能性を伝えるため、`Mutex`は「ポイゾンされた」（\"poisoned\"）状"
```
Comment (kantasv) [inline po/ja.po:14246]: Referred to JA Rustbook's translation, i.e., "poison" -> "ポイゾン" https://doc.rust-jp.rs/rust-nomicon-ja/poisoning.html

**Comment by mgeisler** [comment]
> Note: There are several lines containing outdated `msgid`, but we can refresh the `ja.po` file in a separate MR. We can slightly update corresponding `msgstr` afterwards.

Thanks for thinking of this! It would be great to have the messages refreshed — it would give a new stable foundation for future translations and I don't expect us to rewrite the course again anytime soon.

**Comment by kantasv** [comment]
Hi, @mgeisler yes, I agree. It's a good time to refresh `ja.po` file. I close this MR for now and will get back to this :)

**Pull request closed without merging**